docs/source/build-images.md

```diff
@@ -1,12 +1,32 @@
 # Create a first release and image of your project
 
-This document describes how the [AICoE CI][1] and Thoth pipelines can be used to create the images, and where you can find the images for this project. For the purpose of this tutorial, the required images have already been created using these pipelines.
+This document describes how the [AICoE CI][1] and [Thoth](https://github.com/thoth-station) pipelines can be used to create the images for this project. **For the purpose of this tutorial, we have created the required images of the upstream repo. No action is required.** If you want to build these resources from your own fork, use the following instructions for image builds.
+
+## Set up AICoE CI and Thoth bots on your fork
+
+You will first need to configure your repo to use AICoE CI and Thoth. The [AICoE-CI tooling](https://github.com/AICoE/aicoe-ci) can be set up in just a few steps. Start by installing the AICoE CI GitHub application by [following this link](https://github.com/apps/aicoe-ci). When installing this application, select your profile for the organization, and specify the `elyra-aidevsecops-tutorial` as the repository.
+
+Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. 
+
+<div style="text-align:center">
+<img alt="Invite Sesheta" src="https://raw.githubusercontent.com/thoth-station/elyra-aidevsecops-tutorial/master/docs/images/InviteSesheta.png">
+</div>
+
+The last step is to add the [aicoe-ci configuration file](https://github.com/AICoE/aicoe-ci#aicoe-ci-configuration-file). For the purpose of this tutorial, the [.aicoe-ci.yaml](../../.aicoe-ci.yaml) file is already present, so no further actions are needed to set up the AICoE CI toolkit.
```

Comment:
correct me if I'm wrong @pacospace @harshad16, but do we not need to  configure the `registry-org`, `registry-project` and `registry-secret` for the fork of the repo? The registry project for a fork of the repo will be different from the upstream registry project right?

Also, to set up a registry secret one would have to set up the robot account on the quay repo, correct?

Maybe we should also elaborate on all these additional steps here.

Reply:
Users have to configure those three fields depending on where they want their images, and I totally agree we need to elaborate on how to set up the robot account on the quay repo.
there is some detail in here: https://github.com/AICoE/aicoe-ci#configuring-build-requirements

Reply:
so we should expand a little bit on how to customize the yaml and set up the robot account then. For detailed steps we can point to the aicoe-ci url

Reply:
yes that would be great :)

Reply:
For setting up the robot account, there is a step that says: "Click on the Robot creation, goto kubernetes secret and pass on that information to aicoe-ci instance. Currently you would have to pass it on by contacting us, we are still trying to figure out some alternative." would the "contacting us" be creating an issue, either in this repo or aicoe-ci?